src/shell/subproc.zig

```diff
@@ -845,14 +863,14 @@ pub const ShellSubprocess = struct {
             return .{ .err = .{ .custom = bun.default_allocator.dupe(u8, "out of memory") catch bun.outOfMemory() } };
         };
 
-        spawn_args.env_array.append(allocator, null) catch {
-            return .{ .err = .{ .custom = bun.default_allocator.dupe(u8, "out of memory") catch bun.outOfMemory() } };
-        };
+        // spawn_args.env_array.append(allocator, null) catch {
+        //     return .{ .err = .{ .custom = bun.default_allocator.dupe(u8, "out of memory") catch bun.outOfMemory() } };
```

Comment:
this shouldn't be left commented out

either:
- it should be deleted
- it should be un-commented out if that was a mistake

Reply:
thanks for catching. deleted because the null terminator is inserted by the envp construction.